templates/lp-module/bbscaff.js

```diff
@@ -19,20 +31,35 @@ module.exports = function(bbscaff){
             message: 'Author'
         }
     ], function(answers){
-        bbscaff.fetchTemplate('http://bitbucket.org/backbase/lp-module-blank-template.git', __dirname, function(err){
-            if (err) {
-                return console.error('Error trying to update template from git', err);
-            }
-
-            bbscaff.generate({
-                // LP uses widget.name instead of widget_name
-                module: answers
-            }, {
-                // Sets destination path
-                destination_path: answers.name,
-                // Reset interpolate from bbscaff, so instead of <%=var%> it uses ${var}
-                interpolate: undefined
-            });
-        });
+        checkGithubConnectivity()
+            .then(
+                function (){
```

Comment:
Make prompt return a promise and have it as a part of checkGithubConnectivity instead of repeating?